### ISSUE TITLE
Update README for build/test/push cases

### ISSUE THREAD

**Issue by raladev** [ExchangeUnion/xud-docker#551]

**Comment by raladev** [comment]
Based on https://github.com/ExchangeUnion/xud-docker/pull/546

**Comment by kilrau** [comment]
Correct to point this out and should have been done in the course of #546 , but I think we can do smaller non-programming PRs like this ourselves @raladev

**Comment by raladev** [comment]
> Correct to point this out and should have been done in the course of #546 , but I think we can do smaller non-programming PRs like this ourselves @raladev

Agreed, but according this task I know nothing about new push flags, if they exists and test command.
So, it would be better if developer will update README. Its the same activity like help command updating.

**Comment by kilrau** [comment]
The new `--dev` and `--use-local-images` flags should be added too.
